helm/g2p-sandbox/values.yaml

```diff
@@ -369,8 +369,8 @@ ph-ee-engine:
     hostname: bulk-connector.sandbox.mifos.io
     aws:
       region: "<region>"
-      access_key: "<access_key>"
-      secret_key: "<secret_key>"
+      access_key: "aws-access-key"
```

Comment:
please re-check these values. I think this should be fetched from the secret file

Reply:
This is same as the v1.11.0-release chart